cdhweb/events/templates/events/snippets/event_card.html

```diff
@@ -20,7 +20,7 @@ <h2 property="schema:name">{{ event.title }}</h2>
                 <meta property="schema:name" content="{{ event.location.name }}"/>
                 {% if not event.is_virtual %}
                 <meta property="schema:address" content="{{ event.location.address }}"/>
-                {% elif event.url %}
+                {% elif event.join_url %}
                 <meta property="schema:url" content="{{ event.url }}"/>
```

Comment:
Shouldn't this also be updated to use `event.join_url` instead of `event.url` ?

If we have any unit tests for the templates it would be good to update them with a simple check for this logic.